### PR TITLE
MVP Architectural Pattern

### PR DESCRIPTION
**Pull request by brenpardal**
- Shows the values of the button on the display. 
- Delete and equals functionality are not implemented yet.

showOnDisplayView

### CODE REVIEW

File: app/src/main/java/com/calculator/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        setOnClickListener(binding.buttonNumberZero);
+        setOnClickListener(binding.buttonAdd);
+        setOnClickListener(binding.buttonSub);
+        setOnClickListener(binding.buttonMul);
+        setOnClickListener(binding.buttonDiv);
+        setOnClickListener(binding.buttonComma);
+    }
+
+    private void setOnClickListener(Button button) {
+        button.setOnClickListener(view -> onClickAction(button));
```
Comment (BQuinteros) [inline app/src/main/java/com/calculator/MainActivity.java:47]: Uff Excellent!!

File: app/src/androidTest/java/com/example/calculator/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.example.calculator;
+package com.calculator;
```
Comment (BQuinteros) [inline app/src/androidTest/java/com/example/calculator/ExampleInstrumentedTest.java:1]: I think this class is not necessary you can remove it

File: app/src/main/java/com/calculator/MainActivity.java (hunk 1 above)
Comment (mauroLonghi) [inline app/src/main/java/com/calculator/MainActivity.java:47]: nice!

File: app/src/test/java/com/example/calculator/ExampleUnitTest.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.example.calculator;
+package com.calculator;
```
Comment (BQuinteros) [inline app/src/test/java/com/example/calculator/ExampleUnitTest.java:1]: Same here, is not necessary for you can delete it

File: app/src/main/java/com/calculator/MainActivity.java (hunk 1 above)
Comment (brenpardal) [reply app/src/main/java/com/calculator/MainActivity.java:47]: Thanks!

Comment (brenpardal) [reply app/src/main/java/com/calculator/MainActivity.java:47]: Thanks!

File: app/src/androidTest/java/com/example/calculator/ExampleInstrumentedTest.java (hunk 2 above)
Comment (brenpardal) [reply app/src/androidTest/java/com/example/calculator/ExampleInstrumentedTest.java:1]: Yes! Solved.

File: app/src/test/java/com/example/calculator/ExampleUnitTest.java (hunk 3 above)
Comment (brenpardal) [reply app/src/test/java/com/example/calculator/ExampleUnitTest.java:1]: Yes, solved.

**Review by LuchoElissondoGlobant — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

**Review by LuchoElissondoGlobant — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**